frontend/src/lib/MlmdUtils.ts

```diff
@@ -78,11 +78,11 @@ export enum KfpExecutionProperties {
 }
 
 export const ExecutionHelpers = {
-  getPipeline(execution: Execution): string | number | undefined {
+  getWorkspace(execution: Execution): string | number | undefined {
     return (
-      getResourceProperty(execution, ExecutionProperties.PIPELINE_NAME) ||
+      getResourceProperty(execution, ExecutionCustomProperties.RUN_ID) ||
```

Comment:
I see that `, true` got removed. Was that intended?

Reply:
Yes, without the `true`, it reads from property instead of custom property.
(I should move this to ExecutionProperties later, because it's in kubeflow/frontend repo)

Reply:
Would you mind using
```
getResourceProperty(execution, ExecutionCustomProperties.RUN_ID, true) ||
getResourceProperty(execution, ExecutionCustomProperties.RUN_ID) ||
```
here?
It's a custom_property that's written by MetadataWriter.

P.S. I think we should just change getResourceProperty to always read from both `properties` and `custom_properties`.

Reply:
Agree, I will update this PR